components/hab/tests/test_install_script.bats

```diff
@@ -0,0 +1,100 @@
+setup() {
+  rm -f /bin/hab
+  rm -rf /hab/pkgs/core/hab
+}
+
+darwin() {
+  [ "$(uname)" == "Darwin" ]
+}
+
+linux() {
+  [ "$(uname)" == "Linux" ]
+}
+
+installed_version() {
+  hab --version | cut -d'/' -f1
+}
+
+installed_target() {
+  version_release="$(hab --version | cut -d' ' -f2)"
+  version="$(cut -d'/' -f1 <<< "$version_release")"
+  release="$(cut -d'/' -f2 <<< "$version_release")"
+  cat /hab/pkgs/core/hab/$version/$release/TARGET
+}
+
+@test "Install latest for x86_86-linux" {
+  linux || skip
+  run components/hab/install.sh -c dev
+
+  [ "$status" -eq 0 ]
+  [ "$(installed_target)" == "x86_64-linux" ]
+}
+
+@test "Install specific version for x86_64-linux" {
+  linux || skip
+  run components/hab/install.sh -v 0.89.48 -c dev
```

Comment:
`-c dev` should be removed

Reply:
It would not.  I do think we want to test installations of specific versions, so the final change to this before merge would probably need to be the version we're about to release?  Which seems silly but I can't think of an alternative at this time.

Reply:
After some conversation,  migrating .88 isn't an option just yet.